deploy/charts/validate.sh

```diff
@@ -11,6 +11,9 @@ MIN_KUBE_VERSION="1.23.0"
 if ! command -v kubeconform &> /dev/null; then
     go install github.com/yannh/kubeconform/cmd/kubeconform@latest
 fi
+if ! command -v pluto &> /dev/null; then
+    brew install FairwindsOps/tap/pluto
```

Comment:
Some of us develop on Linux so this has to be something that works cross-platform

Reply:
Can I just display warning to install pluto with the link how to do this? Or do you expect it to install in on Linux and Mac?

Reply:
You're right... I tried it before but something it didn't work :(